src/magicbox/SuggestionsManager.ts

```diff
@@ -184,11 +185,13 @@ export class SuggestionsManager {
 
     if (!this.hasSuggestions) {
       this.removeAccessibilityPropertiesForSuggestions();
+      this.querySuggestPreviewComponent && this.querySuggestPreviewComponent.handleNoSuggestion();
```

Comment:
There's kind of an ugly coupling between SuggestionsManager and the querySuggestPreviewComponent.

I guess for the part where it needs to change it's DOM because it needs to adapt to the query suggest preview component being present is tolerable, but the fact that it also need to call specific function on the `querySuggestPreviewComponent` is not clean.

Could having no suggestions be a case of triggering the "suggestionChanged" event, with an "empty/null" suggestion ?